tests/features/steps/basic_commands.py

```diff
@@ -116,7 +119,8 @@ def step_confirm_destructive_command(context):
         context, 'You\'re about to run a destructive command.\r\nDo you want to proceed? (y/n):', timeout=2)
     context.cli.sendline('y')
 
+
 @then(u'we send password')
 def step_send_password(context):
-    wrappers.expect_exact(context, 'Password for postgres:', timeout=5)
+    wrappers.expect_exact(context, 'Password for', timeout=5)
```

Comment:
expect_exact() only does a `startswith()` check?

Reply:
I think it's looking for a substring in the output. It does not have to match from line start to end.